Platforms/QemuQ35Pkg/Plugins/QemuRunner/QemuRunner.py

```diff
@@ -243,4 +243,12 @@ def Runner(env):
             # Tested same FDs on QEMU 6 and 7, not observing the same.
             ret = 0
 
+        ## TODO: Recover the console. Remove this once QEMU fixes the issue: https://gitlab.com/qemu-project/qemu/-/issues/1674
+        if os.name == 'nt' and qemu_version[0] >= '8':

```

Comment:
Is it worthwhile to do the equivalent on Linux, just in case? `stty sane` will  reset most things.

Reply:
I did have some handling for that around line 234, but that was only for when Linux instance returned failure. Do we really want to apply stty change to all returns?

Reply:
I think I've only seen the Linux terminal get messed up on qemu failure, I was more just thinking of doing a cleanup post qemu, since it does mess with terminal stuff. But I don't care too much one way or another

Reply:
Okay, modified the flow so that it will restore to the original console mode for Windows and `stty sane` for Linux.